### PR TITLE
Removes Dogborg from Module Selection [ADMIN EDIT]

### PR DESCRIPTION
**Pull request by CDRDecky**
should work

if this doesn't i guess the 10 bucks are invalid IDK

### CODE REVIEW

**Comment by IrkallaEpsilon** [comment]
Lyro (Not Lifen) — Today at 1:05 PM
Oh my god why are you making a petition
Comment out line 140 in robot.dm ITS THAT EASY MAKE A PR!!
It literally takes less time to do that than to create a thread about it. Github is also better for discussion on it

FUCK IT BEAT ME TO IT

**Comment by ExoticJazz** [comment]
Follow the proper PR format, put some effort into it and explain how this contributes to the game.

**Review by ExoticJazz — CHANGES_REQUESTED** [review]
Follow the proper PR format, put some effort into it and explain how this contributes to the game.

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 1]
```diff
@@ -137,7 +137,7 @@
 	"Medical" = /obj/item/robot_module/assaultron/medical
 	)
 
-	modulelist += get_cit_modules() //Citadel change - adds Citadel's borg modules.
+    // 	modulelist += get_cit_modules() //Citadel change - adds Citadel's borg modules.
```
Comment (ProbablyCarl) [inline code/modules/mob/living/silicon/robot/robot.dm:140]: Improperly commented.

**Comment by ProbablyCarl** [comment]
There. I got you. That ten dollars is mine, bucko.

**Pull request merged**